package-lock.json

```diff
@@ -1,3 +1,2308 @@
 {
-  "lockfileVersion": 1
+  "name": "mayflower",
```

Comment:
since we are using `yarn` at the root, let's not commit the package-lock file.

Reply:
Removed this package-lock file